core-blocks/image/index.js

```diff
@@ -61,7 +61,7 @@ const blockAttributes = {
 const imageSchema = {
 	img: {
 		attributes: [ 'src', 'alt' ],
-		classes: [ 'alignleft', 'aligncenter', 'alignright', 'alignnone' ],
+		classes: [ 'alignleft', 'aligncenter', 'alignright', 'alignnone', /wp-image-\d+/ ],
```

Comment:
Valid image class: `the-dog-yawp-image-1` ?

Reply:
Ugh, sorry